packages/mdc-textfield/index.js

```diff
@@ -369,6 +369,14 @@ class MDCTextField extends MDCComponent {
     this.foundation_.setUseNativeValidation(useNativeValidation);
   }
 
+  /**
+   * Sets the label element content.
+   * @param {string} content
+   */
+  set labelContent(content) {
+    this.foundation_.setLabelContent(content);
```

Comment:
I suspect we need to update the notched outline's width if it is present and open. Otherwise the label size will update but the notch probably won't by itself. We should at least manually test this to be sure.